snuba/datasets/storages/sessions.py

```diff
@@ -2,7 +2,6 @@
     AggregateFunction,
```

Comment:
I wondered whether to change these in the storage schemas as well

Reply:
> I wondered whether to change these in the storage schemas as well

Which ones are you talking about ?

Reply:
All of the AggregateFunction ones. Instead of the entire `AggregateFunction("quantilesIf(0.5, 0.9)", UInt(32), UInt(8)),` in the storage schema, we'd replace that with `ReadOnly()` or something like that.